diff --git a/admin_menu_source.module b/admin_menu_source.module
index 2e9ef23..96c2f46 100644
--- a/admin_menu_source.module
+++ b/admin_menu_source.module
@@ -58,7 +58,9 @@ function admin_menu_source_admin_menu_output_alter(&$content) {
   // $rid = key(array_reverse($user->roles, TRUE));
 
   // Find the user role rid.
-  $roles_ids = array_keys(user_roles(TRUE, 'access administration menu'));
+  // $roles_ids = array_keys(user_roles(TRUE, 'access administration menu'));
+	// https://drupal.org/node/2202757
+	$roles_ids = array_keys(user_roles(FALSE, 'access administration menu'));
   $user_roles_ids = array_keys($user->roles);
   $user_roles = array_reverse(array_intersect($roles_ids, $user_roles_ids));
   $rid = $user_roles[0];
@@ -120,4 +122,4 @@ function _admin_menu_source_get_role_menu($rid) {
   $settings = _admin_menu_source_get_settings();
   
   return isset($settings[$rid]['source']) ? $settings[$rid]['source'] : '';
-}
\ No newline at end of file
+}
