diff --git a/includes/bootstrap.inc b/includes/bootstrap.inc
index 4cc3914..4393824 100644
--- a/includes/bootstrap.inc
+++ b/includes/bootstrap.inc
@@ -1563,6 +1563,14 @@ function format_string($string, array $args = array()) {
  * @ingroup sanitization
  */
 function check_plain($text) {
+	// Gary
+  if(is_array($text)) {
+    return;
+  }
+  else if(is_object($text)) {
+    return;
+  }
+
   return htmlspecialchars($text, ENT_QUOTES, 'UTF-8');
 }
 
